src/components/AppRouter.js

```diff
@@ -30,10 +31,10 @@ function AppRouter() {
     <BrowserRouter>
       <Switch>
         <DefaultLayout exact={true} path="/" component={Splash} />
+        <DefaultLayout path="/invite_missing" component={InviteMissing} />
```

Comment:
As mentioned in `InviteMissing.js`, I don't really understand what this URL is from its name, at least a comment would be nice.